File: result/translate/transcoder/java2python/python/MINIMUM_INCREMENT_K_OPERATIONS_MAKE_ELEMENTS_EQUAL.py
def f_gold ( arr , n , k ) :
    max = sum ( arr )
    res = 0
    for i in range ( n ) :
        if ( max - arr [ i ] ) % k != 0 :
            return - 1
        else :
            res += ( max - arr [ i ] ) / k
    return res

File: result/translate/starchat/python2java/java/MINIMUM_INCREMENT_K_OPERATIONS_MAKE_ELEMENTS_EQUAL.java
public static int f_gold ( int arr [ ], int n, int k ) {
        int max1 = Integer.MIN_VALUE;
        for ( int i = 0; i < n; i++ ) {
            max1 = Math.max ( max1, arr [ i ] );
        }
        int res = 0;
        for ( int i = 0; i < n; i++ ) {
            if ( ( max1 - arr [ i ] ) % k!= 0 ) {
                return -1;
            }
            res += ( max1 - arr [ i ] ) / k;
        }
        return res;
    }